src/ast.c

```diff
@@ -603,6 +603,13 @@ static jl_value_t *scm_to_julia_(fl_context_t *fl_ctx, value_t e, jl_module_t *m
             temp = (jl_value_t*)jl_exprn(sym, 1);
             jl_exprargset(temp, 0, ex);
         }
+        else if (sym == jl_symbol("anonymous_closure")) {
```

Comment:
Are "anonymous closure" and "opaque closure" the same thing or something different?

Reply:
I'm hoping to get rid of this. I just grabbed for something different from `opaque_closure` since this doesn't do the same thing; it does not create an OpaqueClosure.

Reply:
YAYAKC?